insight/pv_model/pv_model_class.py

```diff
@@ -262,7 +262,7 @@ def _plot(self, x, y, x_label, y_label, title):
 
 def main():
```

Comment:
Do we still need a main method here since this file is mainly just the placeholder for functions used in other files? Or is this main method still useful when testing this file specifically?

Reply:
Good catch! We do not still need the `main` function there, especially since we have functions for all of the important functionalities we need. I don't think we need it for testing either because we have the demo notebooks and will eventually have some sort of `pytest` functions that will test the functions individually. 

I think I might leave it to you to refactor the code and get rid of the `main` function once this PR is submitted (since that change isn't really related to this PR).

Reply:
Ok sounds good!